src/app/profiles/multipleChoiceEditor.tsx

```diff
@@ -0,0 +1,197 @@
+                            }}
+                        >
+                            {({
+                                setFieldValue,
+                            }) => (
+                                <Form>
+                                    <CardHeader>
+                                        <CardTitle>Multiple Choice Question</CardTitle>
+                                        <CardDescription>Order of the answers is tiered and the top one is considered higher/better</CardDescription>
+                                    </CardHeader>
+                                    <CardContent>
+
+                                        <Label>Question Text</Label>
+                                        <Field name="question" type="text" placeholder="" value={questionText} className="flex h-10 w-full rounded-md border border-input bg-background px-3 py-2 text-sm ring-offset-background file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50" onChange={(e: any) => {
+                                            setQuestionText(e.target.value)
+                                            setFieldValue(e.target.name, e.target.value)
+                                        }} />
+
+                                        <DndContext collisionDetection={closestCenter} onDragEnd={onDragEnd}>
+                                            <FieldArray
+                                                name="answers"
+                                                render={arrayHelpers => {
+                                                    ArrayHelperRef.current = arrayHelpers
+                                                    return (
+                                                        <SortableContext items={answers} strategy={verticalListSortingStrategy}>
+                                                            {answers.map((answer, index) => (
+                                                                <SortableInput key={answer.id} answer={answer} index={index} answers={answers} setAnswers={setAnswers} arrayHelpers={arrayHelpers} setFieldValue={setFieldValue} />
+                                                            ))}
+                                                        </SortableContext>
+                                                    )
+                                                }}>
+                                            </FieldArray>
+                                        </DndContext>
+                                    </CardContent>
+                                    <CardFooter className="flex justify-between">
+                                        <Button type="button" onClick={() => {
+                                            setAnswers([...answers, {id: answers.length + 1, label: ''}])
```

Comment:
not a fan of `answers.length + 1`, not a strong way to guarantee uniqueness, assuming that's the goal. Not to mention, I'm not entirely sure what we're using the ids of each answer for?